### ISSUE TITLE
$FILE Not working on Windows, because of the prepended /

### ISSUE THREAD

**Issue by B3ret** [Vehmloewff/custom-format#1]
I'm trying to get qmlformat to run on Windows.

See image.

![grafik](https://user-images.githubusercontent.com/7371857/120073895-a0311580-c09a-11eb-9ca2-bc1ae992a44d.png)

**Comment by B3ret** [comment]
Seems to be unrelated to that slash
My bad

**Comment by DiazMichael** [comment]
@B3ret did you manage to fix it? I am having a similar issue.

**Comment by B3ret** [comment]
> 
> 
> @B3ret did you manage to fix it? I am having a similar issue.

I didn't get it to work. I'm using a script solution now.

**Comment by Vehmloewff** [comment]
Thanks for the input @B3ret.  I'll have to look into this.

This tool was never actually tested on windows because I didn't have access to a windows device at the time of writing.  I do, however, have access to one now.

I have no idea what would be causing this issue because the `filepath` custom-format uses to replace `$FILE` is the vscode provided `document.uri.path`, which should be windows compatible on windows.  I will, however, try to get to the bottom of this issue.

**Comment by alaestor** [comment]
I'm having a similar issue;
```
[error] Received an error while running the following format command on /c:/Users/John Doe/Desktop/dev/snippets/IntegralRange.hpp:
> python.exe "C:/Program Files/Test/FGL_CustomFormat.py" "$FILE"
Error:
> Command failed with ENOENT: python"C:/Program Files/Test/CustomFormat.py" "/c:/Users/John Doe/Desktop/dev/snippets/IntegralRange.hpp"
> spawn python.exe ENOENT
```

I thought perhaps it was having an issue with finding `python.exe` but when I provide an absolute path

```
> Command failed with ENOENT: "C:/Program Files/Python39/python.exe" "C:/Program Files/Test/CustomFormat.py" "/c:/Users/John Doe/Desktop/dev/snippets/IntegralRange.hpp"
> spawn C:\WINDOWS\system32\cmd.exe ENOENT
```

An interesting derp. Sadly I don't know enough about TS to contribute to the debugging effort...

Good luck lol

**Comment by minism** [comment]
I took a look and this seems to be fixed by https://github.com/Vehmloewff/custom-format/pull/2

The correct property seems to be `document.uri.fsPath`

https://code.visualstudio.com/api/references/vscode-api

**Comment by Vehmloewff** [comment]
Fixed by #2
